processes/issue-triage.md

```diff
@@ -0,0 +1,95 @@
+        1. if you do not understand, ask the submitter questions to clarify
+        1. if you're not sure which questions to ask, ask a reviewer for assistance and leave the `carvel triage` label in place
+
+### 2. Triage Issues by Type
+For all issues, `kind` labels are generally supplied by the submitter. Ensure that the correct label is applied and update it when necessary.
+
+#### Enhancement
+1. Read the issue and try to understand the ask.
+1. If information is missing or something is not clear, reply to the issue asking for further clarification. Add the `triage/needs-more-information` label. You should keep tabs on this issue until triage is complete.
+1. If the issue looks like a good improvement for the tool,
+  1. add a comment explaining your reasoning
+  1. define its priority
+  1. change the label to `carvel accepted` from `carvel triage`
+  1. if appropriate, add the `good first issue` label
+1. If the issue does not look like a good fit for the tool, add a comment explaining your reasoning and close the issue.
+
+#### Bug
+1. Try to replicate the issue.
+1. If you're able to replicate the issue,
+  1. Define its priority
+  1. change the label to `carvel accepted` from `carvel triage`
+1. If you're unable to replicate the issue,
+  1. add the `triage/not-reproducible` label
+  1. ask the submitter for more information
+  1. if both parties agree that the issue can't be reproduced then close the issue
+
+### 3. Define Priority
+We use labels for prioritization. If an issue lacks a priority label then it has not been reviewed and prioritized completely, yet.
+
+We aim for consistency across the entire project. If you notice an issue that you believe to be incorrectly prioritized, please leave a comment offering your counter-proposal and we will evaluate it.
+
+| Priority label | What it means | Examples |
+|---|---|---|
+| `priority/0` | Team leaders are responsible for making sure that these issues (in their area) are being actively worked on (i.e., drop what you're doing). These should be fixed before the next release. | user-visible bugs in core features <br> broken builds or tests <br> critical security vulnerabilities |
+| `priority/1` | Must be staffed and worked on either currently or very soon. Ideally, this will be done in time for the next release. Important, but wouldn't block a release. | Work to consider for the current or next release  |
+| `priority/2` | Important over the long term, but may not be currently staffed and/or may require multiple releases to complete. Wouldn't block a release. | Work to consider for the roadmap (1+ quarters out) |
+| `priority/3` | General agreement that this is a nice-to-have, but no one is allocated to work on it anytime soon. |  |
+| `priority/4` | Possibly useful but not yet enough support to actually incorporate it. | Placeholders for potentially good ideas so that they don't get completely forgotten. |
```

Comment:
@gcheadle-vmware You mentioned that the `priority/x` system wasn't intuitive. Looking at [how k8s does this](https://github.com/kubernetes/community/blob/master/contributors/guide/issue-triage.md#step-three-define-priority), they use the following:
1. priority/critical-urgent
2. priority/important-soon
3. priority/important-longterm
4. priority/backlog
5. priority/awaiting-more-evidence

Do you have a preference for one versus the other? Or do you have any other suggestions?

Reply:
I made one update to #4 and changed it to `priority/unprioritized-backlog`.